TESTS/mbedtls/ecp/main.cpp

```diff
@@ -20,8 +20,6 @@
 #include "utest.h"
 #include "rtos.h"
 
-using namespace utest::v1;
-
```

Comment:
It is not a good idea to use the ```using ``` statement as in future the test case might want to use the other ```status_t``` type. And it would be ambiguous.